defaults/main.yml

```diff
@@ -78,3 +78,12 @@ apache_modjk_workers_properties_file: # Name and extension of this file
 ## logrotate variables
 apache_logrotate_enabled: true
 apache_logrotate_cron_settings: {hour: '3', minute: '0', user: 'root'}
+
+## brotli variables
+apache_brotli_install: false
+apache_brotli_lib_path: /usr/lib/brotli
+apache_brotli_build_path: /tmp/brotli
+apache_brotli_version: v1.0.2
+apache_brotli_prerrequisites:
```

Comment:
Maybe this variable should be at vars

Reply:
Done

Reply:
For some reason, when using Vagrant, `build-essential` is required. If this list is in `vars.yml`, its precedence is too high and is not that comfortable to overwrite it in the playbook.